### PR TITLE
Core: Load the onerror module

### PR DESCRIPTION
**Pull request by leobalter**
Ref gh-1062

This is not fixing #1062, but it fixes another bug I found from there.

### CODE REVIEW

**Comment by leobalter** [comment]
No, @jquerybot, it's not my first contribution to this project. :)

**Comment by platinumazure** [comment]
I'd better track this one while thinking about #1061. To me, it's a bit of a smell that we are importing that file for its side effects- better that that module export an error handling function and the HTML reporter use that to create a `window.onerror` handler.

**Comment by leobalter** [comment]
I believe it's time to release a new version, this one can land faster, but it shouldn't block your work on #1061

**Comment by platinumazure** [comment]
No objection to releasing this in a new version, #1061 still needs
discussion and buy-in. Thanks!

File: src/core.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import config from "./core/config";
 import { defined, extend, objectType, is, now } from "./core/utilities";
 import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
+import {} from "./core/onerror";
```
Comment (trentmwillis) [inline src/core.js]: You be able to just do:

```
import "./core/onerror";
```

Similar to what is done in `src/qunit.js`. I prefer this pattern as it is a good indicator that you simply want the module evaluated, but don't expect exports from it.

**Comment by trentmwillis** [comment]
Went ahead and made the adjustment myself and merged.

**Pull request closed without merging**